Makefile

```diff
@@ -30,8 +30,8 @@ LDFLAGS := -X $(REPOPATH)/pkg/version.version=$(VERSION) \
 	-X $(REPOPATH)/pkg/version.openshiftVersion=$(OPENSHIFT_VERSION) -s -w -extldflags '-static'
 
 GOFILES := go list  -f '{{join .Deps "\n"}}' ./cmd/minikube/ | grep $(REPOPATH) | xargs go list -f '{{ range $$file := .GoFiles }} {{$$.Dir}}/{{$$file}}{{"\n"}}{{end}}'
-GOFILES_NO_VENDOR := $(GOFILES) | grep -v /vendor
 PACKAGES := go list ./... | grep -v /vendor
+SOURCE_DIRS = cmd pkg
```

Comment:
@hferentschik , I think `test` directory should be included. It contains integration tests.